Makefile

```diff
@@ -28,6 +28,11 @@ VERSION       = $(shell $(PYTHON) setup.py --version)
 
 TAG           = v$(VERSION)
 
+COVERAGE=coverage
+ifeq ($(PYTHON),python3)
+  COVERAGE=coverage3
```

Comment:
Please don't indent this line inside the ifeq/endif block.  GNU make can get a little irritable with indentation.